query-model/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/expression/ExpressionsTest.kt

```diff
@@ -467,6 +468,20 @@ class ExpressionsTest : WithAssertions {
         assertThat(actual).isEqualTo(expected)
     }
 
+    @Test
+    fun mod() {
+        // when
+        val actual = Expressions.mod(intExpression2, intExpression1)
```

Comment:
Is there a reason why the numbers in intExpression are reversed?

Reply:
I assumed intExpression1, intExpression2 is correct order, because intExpression2 value is bigger than intExpression1.

Reply:
I think `Expressions.mod(intExpression1, intExpression2)` is better because the mod operation doesn't always require the left operand to be greater than the right operand, so I prefer code that doesn't make the reader wonder about intent when looking at the test code.

Reply:
Reflected now.